src/main/resources/db/changelog/changes/02-add-books.yaml

```diff
@@ -0,0 +1,47 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-books
```

Comment:
I would not hardcode any books, we can create some admin users like that but some usual entity I would not do in such a way